src/oracle.c

```diff
@@ -1141,21 +1147,10 @@ main(int argc, char **argv)
 			fatal("Unsupported RSA bits: %s\n", opt_rsa_bits);
 	}
 
-	if (!opt_key_format || !strcasecmp(opt_key_format, "raw"))
-		tpm2key_fmt = false;
-	else
-	if (!strcasecmp(opt_key_format, "tpm2.0"))
-		tpm2key_fmt = true;
-	else
-		fatal("Unsupported key format \"%s\"\n", opt_key_format);
-
-	if (!opt_policy_format || !strcasecmp(opt_policy_format, "grub2"))
-		systemd_json = false;
-	else
-	if (!strcasecmp(opt_policy_format, "systemd"))
-		systemd_json = true;
-	else
-		fatal("Unsupported policy format \"%s\"\n", opt_policy_format);
+	if (opt_target_platform == NULL)
+		opt_target_platform = "oldgrub"; /* we should probably change this to tpm2.0 now */
```

Comment:
I agree with the comment. The "oldgrub" or "raw" format is not used by fde-tools anymore, and the shim-install script only expects "tpm2.0" now.